modules/rosa-upgrading-manual-ocm.adoc

```diff
@@ -9,13 +9,13 @@ endif::[]
 
 :_content-type: PROCEDURE
 [id="rosa-upgrade-ocm_{context}"]
-= Upgrading using the console
+= Scheduling individual upgrades through the {cluster-manager} console
 
-You can upgrade a {product-title} cluster
+You can schedule to upgrade a {product-title} cluster
```

Comment:
s/schedule to upgrade a/schedule upgrades for a

Reply:
Updated.